### PR TITLE
Add fixes.

### PR DESCRIPTION
**Pull request by Andrei-Dolgolev**

### CODE REVIEW

File: backend/moonstreamapi/data.py [hunk 1]
```diff
@@ -252,9 +252,9 @@ class DashboardResource(BaseModel):
...
 class DashboardCreate(BaseModel):
     name: str
-    subscriptions: List[DashboardMeta]
+    dashboard_subscriptions: List[DashboardMeta]
 
 
 class DashboardUpdate(BaseModel):
     name: Optional[str]
-    subscriptions: List[DashboardMeta] = Field(default_factory=list)
+    dashboard_subscriptions: List[DashboardMeta] = Field(default_factory=list)
```
Comment (zomglings) [inline backend/moonstreamapi/data.py:260]: `subscription_settings`

**Review by zomglings — CHANGES_REQUESTED** [review]
`dashboard_subscriptions` -> `subscription_settings`

Also make the appropriate change on frontend.

File: backend/moonstreamapi/data.py (hunk 1 above)
Comment (Andrei-Dolgolev) [reply backend/moonstreamapi/data.py]: done

**Comment by Andrei-Dolgolev** [comment]
required run resource migration 
`python -m moonstreamapi.admin.cli migrations run --id 20211202`
rename `dashboard_subscriptions` --> `dashboard_settings`.

**Pull request closed without merging**